contracts/bridge/Bridge.go

```diff
@@ -525,6 +525,32 @@ func (_Bridge *BridgeCallerSession) HandleNonce() (uint64, error) {
 	return _Bridge.Contract.HandleNonce(&_Bridge.CallOpts)
 }
 
+// HandledRequestTx is a free data retrieval call binding the contract method 0x8a75eee2.
+//
+// Solidity: function handledRequestTx( bytes32) constant returns(bool)
```

Comment:
```suggestion
// Solidity: function handledRequestTx(bytes32) constant returns(bool)
```
Please, correct the same typo in other lines also.

Reply:
The go file is automatically generated by abigen. But I will try to find how to fix it.

Reply:
@aidan-kwon This template code is in `github.com/klaytn/klaytn/accounts/abi/bind/template.go`.
I can't find the solution now. I will consider it later. :)

Reply:
@KimKyungup Thank you. I resolve the conversation.